src/components/HoldPages/PatronIneligibilityErrors.tsx

```diff
@@ -0,0 +1,73 @@
+import { Text, Box, List } from "@nypl/design-system-react-components"
+
+import type {
+  HoldErrorStatus,
+  PatronEligibilityStatus,
+} from "../../types/holdPageTypes"
+import type Item from "../../models/Item"
+import RCLink from "../Links/RCLink/RCLink"
+import ExternalLink from "../Links/ExternalLink/ExternalLink"
+import { PATHS } from "../../config/constants"
+import { appConfig } from "../../config/config"
+
+interface PatronIneligibilityErrorsProps {
+  patronEligibilityStatus: PatronEligibilityStatus
+}
+
+/**
+ * The PatronIneligibilityErrors component is responsible for rendering a list of reasons why
+ * a patron is ineligible to place holds within the HoldRequestErrorBanner component.
+ */
+const PatronIneligibilityErrors = ({
+  patronEligibilityStatus,
+}: PatronIneligibilityErrorsProps) => {
+  const { expired, moneyOwed, ptypeDisallowsHolds, reachedHoldLimit } =
+    patronEligibilityStatus
+
+  const hasSpecificReason =
+    expired || moneyOwed || ptypeDisallowsHolds || reachedHoldLimit
+
+  // Generic patron error displayed in heading, don't show reasons list if there isn't one
+  if (!hasSpecificReason) return null
+
+  return (
+    <Box mt="xs">
+      <Text mb="xs">This is because:</Text>
+      <List
+        type="ul"
+        margin={0}
+        listItems={[
+          ...(expired
```

Comment:
I think this awkward array construction can be avoided if you pass the children in explicitly instead of as ListItems. eg 
```
<List type="ul" margin={0}>
        {expired && (
          <>
            Your account has expired -- Please see{" "}...
```